File: code/codes/js/remove_comment.js
const ACCESS_TOKEN = "발급받은 ACCESS_TOKEN 입력";
const HOST = "https://openapi.band.us";
const API = "/v2/band/post/comment/remove";

// parameters
const BAND_KEY = "검색할_밴드_키";
const POST_KEY = "검색할_포스트_키";
const COMMENT_KEY = "검색할_댓글_키";

const xhr = new XMLHttpRequest();

xhr.onload = function () {
  if (xhr.status != 200) {
    // Error!
    return;
  }
  var responseObj = xhr.response;
  console.log(responseObj); // response!
};
xhr.open(
  "POST",
  `${HOST}${API}?access_token=${ACCESS_TOKEN}&band_key=${BAND_KEY}&post_key=${POST_KEY}&comment_key=${COMMENT_KEY}`
);
xhr.setRequestHeader("Content-Type", "application/json");
xhr.responseType = "json";
xhr.send();

File: code/codes/ruby/remove_comment.rb
require 'pp'
require_relative '0_env'

# https://developers.band.us/develop/guide/api/remove_comment

# 댓글 삭제
# 특정 댓글을 삭제합니다. 동일한 Client ID에서 작성된 댓글을 삭제할 수 있습니다.

# コメント削除
# 特定のコメントを削除。 
# 同じClient IDが作成したコメントを削除できます。

params = {
    'band_key' => 'INPUT YOUR BAND KEY',
    'post_key' => 'INPUT YOUR POST KEY',
    'comment_key' => 'INPUT YOUR COMMENT KEY'
}
response = post_request('https://openapi.band.us/v2/band/post/comment/remove', params)
pp response

result_data = response['result_data']
properties = ['message']
puts validate?(result_data, properties)

message = result_data['message']
raise "NOT SUCCESS" unless message == 'success'